lua/tardis/screens/cl_screen_music.lua

```diff
@@ -1,5 +1,38 @@
 -- Music
 
+local sounds={
+    {"Main Theme (2005)", "theme1"},
+    {"Main Theme (2009)", "theme2"},
+    {"Main Theme (2010)", "theme3"},
+    {"Main Theme (2013)", "theme4"},
+    {"Ninth Doctor", "nine"},
+    {"Tenth Doctor", "ten"},
+    {"Eleventh Doctor", "eleven"},
+    {"Rose Tyler", "rose"},
+    {"Martha Jones", "martha"},
+    {"Donna Noble", "donna"},
+    {"Amy Pond", "amy"},
+    {"River Song", "river"},
+    {"Clara Oswald", "clara"},
+    {"Abigail's Song", "abigail"},
+    {"This is Gallifrey", "thisisgallifrey"},
+    {"Gallifrey", "gallifrey"},
+    {"Vale Decem", "valedecem"},
+    {"The Majestic Tale", "majestictale"},
+    {"Forgiven", "forgiven"},
+    {"The Wedding of River Song", "weddingofriversong"},
+    {"All the Strange Creatures", "allthestrangecreatures"},
+    {"You're Fired", "yourefired"},
+    {"Whose Enigma", "whoseenigma"},
+    {"The Long Song", "thelongsong"},
+    {"Infinite Potential", "infinitepotential"},
+    {"The New Doctor", "thenewdoctor"},
+    {"My Husband's Home", "myhusbandshome"},
+    {"Doomsday", "doomsday"},
+    {"Dark and Endless Dalek Night", "darkandendlessdaleknight"},
+    {"The Greatest Story Never Told", "greateststorynevertold"},
+}
+
 TARDIS:AddScreen("Music", {id="music", menu=false, order=10}, function(self,ext,int,frame,screen)
```

Comment:
The dropdown list does not work in the 3D2D view due to an implementation issue, so I'd recommend that we now force this screen to popup much like the destination screen (coordinate input). You can do this by adding popuponly = true here.

```suggestion
TARDIS:AddScreen("Music", {id="music", menu=false, order=10, popuponly=true}, function(self,ext,int,frame,screen)
```